labelbox/client.py

```diff
@@ -1704,3 +1704,27 @@ def unarchive_feature_schema_node(self, ontology_id: str,
             raise labelbox.exceptions.LabelboxError(
                 "Failed unarchive the feature schema node, message: ",
                 response.text)
+
+    def get_batch(self, project_id: str, batch_id: str) -> Entity.Batch:
+        # obtain batch entity to return
+        get_batch_str = """query %s($projectId: ID!, $batchId: ID!) {
+                          project(where: {id: $projectId}) {
+                             batches(where: {id: $batchId}) {
+                                nodes {
+                                   %s
+                                }
+                             }
+                        }
+                    }
+                    """ % ("getProjectBatchPyApi",
+                           query.results_query_part(Entity.Batch))
```

Comment:
I'll leave the decision on this to your team - and follow existing patterns here for now to maintain code consistency

Reply:
@mnoszczak chatted with Richard about this offline. We'll go ahead with GQL APIs for now, since airbnb needs this ASAP, and it's a sizable lift to convert the existing APIs to REST. Orch team will convert these to REST in the future